tests/FSharp.Test.Utilities/Compiler.fs

```diff
@@ -1331,13 +1331,15 @@ module rec Compiler =
         if not (actual |> Array.contains expected) then
             failwith ($"The following signature:\n%s{expected}\n\nwas not found in:\n" + (actual |> String.concat "\n"))
 
-    let printSignatures cUnit =
+    let private printSignaturesImpl pageWidth cUnit  =
         cUnit
         |> typecheckResults
-        |> signatureText
+        |> signatureText pageWidth
         |> string
         |> fun s ->
             s.Replace("\r", "").Split('\n')
             |> Array.map (fun line -> line.TrimEnd())
             |> String.concat "\n"
-            |> fun tap -> tap
\ No newline at end of file
+    
+    let printSignatures cUnit = printSignaturesImpl None cUnit
+    let printSignaturesWith pageWidth cUnit = printSignaturesImpl (Some pageWidth) cUnit
```

Comment:
Maybe this should be named `printSignaturesWithPageWidth` or `printSignaturesWithWidth` so it's more obvious at the call site

Reply:
I can do this in a future PR. I'd like to have this one merged in as it contains some infrastructure that will be useful to tackle the next slew of problems.

Reply:
Sure, it's not critical :)